datadog.conf.example

```diff
@@ -97,17 +99,6 @@ gce_updated_hostname: yes
 # histogram_aggregates: max, median, avg, count
 # histogram_percentiles: 0.95
 
-# Multiple endpoints/api_keys
-# If you want to send your data to multiple accounts,
-# add your extra api_keys here in a comma-separated list
-# (default: disabled)
-# other_api_keys: my_apikey, my_other_apikey
-# If you want to send your data to multiple endpoints,
-# add your extra api_keys here in a comma-separated list.
-# For each extra dd_url, an api_key must be specified in
-# other_api_keys (default: disabled)
-# other_dd_urls: https://app.example.com
```

Comment:
I don't think so since we're the only ones likely to use it. I only put it there because it was illogical to explain `other_api_keys` without explaining `other_dd_urls`